README.md

```diff
@@ -1 +1,21 @@
-# tibbletest
\ No newline at end of file
+Test Example
+================
+
+``` r
+library(tibbletest)
+
+descriptives(example_dat, "treat", c("age", "sugar_factor", "gender", "happiness", "happy"))
+```
+
+    ## # A tibble: 9 x 4
+    ##   Variable     Label  candy         `ice cream`  
+    ##   <chr>        <chr>  <chr>         <chr>        
+    ## 1 gender       female 124 (48.25%)  110 (45.27%) 
+    ## 2 gender       male   133 (51.75%)  133 (54.73%) 
+    ## 3 happiness    happy  185 (76.76%)  181 (78.7%)  
+    ## 4 happiness    sad    56 (23.24%)   49 (21.3%)   
+    ## 5 happy        no     56 (23.24%)   49 (21.3%)   
+    ## 6 happy        yes    47 (19.5%)    44 (19.13%)  
```

Comment:
shouldn't the table combine `happy = "yes"` and `happy = "Yes"` into the same category?

Reply:
Probably. In `methodsR` there is a `str_to_lower` when converting to logical (only relevant for yes/no). 

If we'd like this to be a feature in general, we'd have to decide what the output should be. How do we decide whether to convert them all to "Yes" or "yes" in a generalizable way?

Reply:
That's what I assumed would be used here, but it would also make sense for that to just be cleaned up in a mutate prior to using the function.  I think especially in this example, we should show how we want the user to handle cases like this (either prior cleaning, or the function collapses this special case).  If we do want to automate this, they should probably all be "Yes" if they are not collapsed to logical output since it probably wouldn't be reported all lowercase in report tables.

Reply:
I kept this version very basic to keep the code-base more straightforward, but I do want to implement the automatic conversion to logical. There is an issue for it #5. 

The yes/no case will be taken care of by the logical conversion, but what do we do in other situations? Let's say for medical procedure type there are values "OPEN", "Open", "open", and "laparoscopic". What do we do with the three opens?

I don't know if there is a good generalizable way to decide which is better, since it'll depend on how all the other labels will be formatted. Most words will look better capitalized, but what about acronyms? 

I think collapsing them is a worthwhile feature, and it doesn't really matter which we collapse to. The first version listed or whichever is first when sorting? Should we add this as an issue, to just keep this first MR with the very basic functionality?

Reply:
I think for cases other than yes/no the user should clean that up prior to running the function.  Too many edge cases that could pop up, and it is more of a data cleaning issue (the user knows what label they will want to use in each case) than a problem to fix when creating a summary table in my opinion.  I think we should only really implement #5 in the function.  It just caught my  eye since it was in the README and wanted to confirm what the goal was for dealing with that.

Reply:
I think it's a nice feature to have to avoid having to do that common data cleaning step, but it could cause problems if it hides the issue from the user and they later use the data assuming that there was no issue. If we did implement this as a feature, we should give a message saying what labels were collapsed. 

Which edge cases are you envisioning? Where the user would want two labels with the only difference being the case to be treated differently?

Reply:
Yes a message should be returned if any labels are collapsed by the function.  I'm thinking typos or misspellings of a label in the data.  Differences in capitalization is easy to deal with and standardize in the function, but typically that's not the only case in messy data.

Reply:
Would it make sense to make a `preprocessing` function that does variable name cleaning (`janitor::clean_names`?), label cleaning, and conversion to logical if the user wants them? These are things that should be done to change the data, not in the table creation step

Reply:
I like that idea, I use `janitor::clean_names()` almost every time I read data in now, and it makes sense to do all of the data manipulation prior to running the function to create the table.  If we could somehow implement our `var_mapping()` function into that as well, that'd be ideal.

Reply:
I was thinking about `var_mapping()`, would something like that go in the `format` function somehow?

Reply:
It might make more sense in the `format()` function, we just have to think about how the user could input their preferred variable names for the table